tests/integration/test_hyper_transformer.py

```diff
@@ -199,6 +200,19 @@ def test_hypertransformer_default_inputs():
             actual = reverse_transformed.iloc[row, column]
             assert pd.isna(actual) or expected == actual
 
+    assert isinstance(ht.field_transformers['integer'], FloatFormatter)
+    # assert ht.field_transformers['integer'].get_output_columns() == ['integer.value']  noqa
```

Comment:
@pvk-developer  I can't test the code being commented out yet, because it needs code from the other PR's I'm writing. Leaving it here so I can uncomment it after rebasing in the other PR.